src/otx/core/data/dataset/segmentation.py

```diff
@@ -64,15 +64,15 @@ def __init__(
             image_color_channel,
             stack_images,
         )
-        self.meta_info = SegMetaInfo(
-            label_names=self.meta_info.label_names,
-            label_groups=self.meta_info.label_groups,
+        self.label_info = SegMetaInfo(
```

Comment:
Should this be renamed into `SegMetaInfo -> SegLabelInfo`?

Reply:
Done, https://github.com/openvinotoolkit/training_extensions/pull/2994/commits/1e9da826b42e4f1bb6ad08cfdd493c076fe02d2e. I also changed `HLabelMetaInfo --> HLabelInfo` and previous `HLabelInfo` --> `HLabelData`

Reply:
Why do we need `HLabelData` in addition to `HlabelInfo`? If not necessary, can we merge them in the next PR?

Reply:
Sure, I'll merge at the next PR